experimental/generated/beats/fields.ecs.yml

```diff
@@ -3919,20 +3942,57 @@
         instrumentation.
       example: openai
       default_field: false
+    - name: system_instructions
+      level: extended
+      type: flattened
```

Comment:
Note: Some of the points I brought up inhttps://github.com/elastic/ecs/pull/2532#issuecomment-3434285628 will apply to flattened as well.